doc/tutorials/electrodes/electrodes_part1.ipynb

```diff
@@ -0,0 +1,500 @@
+    "In metallic confinement, the ion pair interaction energy is screened\n",
+    "exponentially due to the presence of induced charges on the slit walls.\n",
+    "<a href='#[1]'>[2]</a>  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Theoretical Background \n",
+    "\n",
+    "The normal component of electric field across a surface dividing two dielectric\n",
+    "media yields a discontinuity, which can be expressed in terms of a finite\n",
+    "surface charge density \n",
+    "$(\\epsilon_1\\vec{E}_1 - \\epsilon_2\\vec{E}_2).\\hat{n}=-\\sigma(\\vec{r})$.\n",
+    "This expression describes the jump in the electric field across the material\n",
+    "interface going from a dielectric medium $\\epsilon_1$ to another one,\n",
+    "$\\epsilon_2$.\n",
+    "\n",
+    "While in the case of non-polarizable materials ($\\epsilon_1 = \\epsilon_2 = 1$),\n",
+    "this jump is only related to surface charges and dielectric contrast and the\n",
+    "potential is continuous across the interface, for polarizable materials also the\n",
+    "polarization field $\\vec{P}$ will give a contribution. \n",
+    "In order to solve the problem in presence of a jump of the dielectric constant\n",
+    "across an interface, one must know the electric fields on both sides. \n",
+    "\n",
+    "Another approach is to replace this two domain problem by an equivalent one\n",
+    "without the explicit presence of the dielectric jump.\n",
+    "This is achieved by introducing an additional fictitious charge i.e. an induced\n",
+    "charge density $\\sigma_{ind}$ on the surface. \n",
+    "With this well known \"method of image charges\", it is sufficient to know the\n",
+    "electric field on one side of the interface. \n",
+    "**ESPResSo** provides the \"Induced Charge Calculation with fast Coulomb Solvers\"\n",
+    "*(ICC  $\\star$) algorithm <a href='#[1]'>[1]</a> which employs a  numerical\n",
+    "*scheme for solving the boundary integrals and the induced charge.   \n",
+    "\n",
+    "*Note*: Apart from ICC $\\star$, **ESPResSo** offers the \"Electrostatic layer\n",
+    "correction with image charges\" (ELC-IC) method <a href='#[3]'>[3]</a>, for\n",
+    "planar 2D+h partially periodic systems with dielectric interfaces.\n",
+    "The tutorial on *Basic simulation of electrodes in ESPResSo part II*\n",
+    "addresses this in detail.\n",
+    "\n",
+    "The Green's function for two point charges in a dielectric slab can be solved\n",
+    "analytically <a href='#[2]'>[2]</a>.\n",
+    "In the metallic limit the dielectric contrast is\n",
+    "$$ \\Delta = \\frac{\\epsilon_1 - \\epsilon_2} {\\epsilon_1 + \\epsilon_2} = -1 .$$\n",
+    "If the ions are placed in the center of the slab of width $w$ and a distance $r$\n",
+    "away from each other, the Green's function accounting for all image charges\n",
+    "simplifies to <a href='#[4]'>[4]</a> \n",
```

Comment:
mention somewhere (in a heading) that the calculation of this greens function is the main point of the tutorial